Cargo.toml

```diff
@@ -403,7 +403,7 @@ uucore_procs = { version = "0.5.0", package = "uucore_procs", path = "src/uucore
 uu_ls = { version = "0.5.0", path = "src/uu/ls" }
 uu_base32 = { version = "0.5.0", path = "src/uu/base32" }
 uutests = { version = "0.5.0", package = "uutests", path = "tests/uutests" }
-
+string-interner = "0.19.0"
```

Comment:
this isn't the right place to declare it

Reply:
I'm sorry, where is that?